### ISSUE TITLE
Use notcurses to improve graphics

### ISSUE THREAD

**Issue by WSLUser** [eramdam/pokemonshow#12]
Display far superior images in the terminal by using notcurses: https://github.com/dankamongmen/notcurses

**Comment by eramdam** [comment]
If there's a nodejs library to take advantage of that I can try 😁

**Comment by WSLUser** [comment]
There's plenty of wrappers and think there's one you could use with node but hopefully you've been able to check yourself by now

**Comment by eramdam** [comment]
> There's plenty of wrappers and think there's one you could use with node but hopefully you've been able to check yourself by now

I haven't 😅

**Comment by eramdam** [comment]
> There's plenty of wrappers and think there's one you could use with node but hopefully you've been able to check yourself by now

I haven't 😅 And as far as I can tell there doesn't seem to be anything for Node? 
- https://github.com/dankamongmen/notcurses#wrappers
- https://www.npmjs.com/search?q=notcurses

**Comment by WSLUser** [comment]
Hmmm, someone well versed in Node could probably add a wrapper. Also some aren't part of the main repo and maintain thier own fork. For example there's wrappers for Nim and Go not included in the main repository so might be node wrapper has own fork or might need to be made still.

**Comment by eramdam** [comment]
Yeah... I figure maybe that's a use-case for native Node modules which i think can be C++ code? I'm way out of my depth so unless I get the time to learn/mess with that stuff I'd rather wait for a wrapper 🙈
